sql.txt

```diff
@@ -1,14 +1,40 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+    SELECT CustomerName, ContactName, Country 
```

Comment:
Almost! For the `LIMIT` you want to use 10 instead of 20 because the range 11 through 20 is a total of 10 entries. `20-11 + 1 = 10`